pkg/asset/manifests/template.go

```diff
@@ -1,5 +1,31 @@
 package manifests
 
+// AwsCredsData is used to generate cloud-creds-secret
+type AwsCredsData struct {
+	Base64encodeAWSaccessKeyID     string
+	Base64encodeAWSsecretAccessKey string
+}
+
+// OpenStackCredsData is used to generate cloud-creds-secret
+// TODO: Fill this
+type OpenStackCredsData struct {
+	Base64encodeOpenStackSomething     string
+	Base64encodeOpenStackSomethingElse string
+}
+
+// LibvirtCredsData is used to generate cloud-creds-secret
+// TODO: Fill this
+type LibvirtCredsData struct {
+	Base64encodeLibvirtSomething     string
+	Base64encodeLibvirtSomethingElse string
+}
+
+type cloudCredsTemplateData struct {
+	AwsCredsData
```

Comment:
Give explicit names to these fields (e.g., AWS, OpenStack, Libvirt). That will (1) make the name in the template file shorter and (2) allow for fields in the various cred data struct to have the same names.

Reply:
done